src/afl-framework/libexecutors.so.c

```diff
@@ -0,0 +1,16 @@
+#include "afl-fuzz.h"
```

Comment:
will there be multiple libraries or just one? if there is just one then this should be called libafl++.so.c IMHO

Reply:
This I'm not sure about yet. We're supposed to have executors, mutators, seed schedulers etc, so having different libraries for them, is it a good idea?

Reply:
hmm If they can be independent than we could have different libraries. if however a feature requires features from other libraries then splitting them up provides no value IMHO.
Maybe we first have only one library and if it makes sense to split it up this is future work? @domenukk ? @hexcoder- ?

Reply:
I would use extra libraries for optional things (which need not to be loaded every time). But that would be an optimization, so having one library at first seems reasonable.